DIY-Peach!.md

```diff
@@ -0,0 +1,19 @@
+Guide for growing a Peach
+====
+
+As inspired by [Patterns for Decentralized Organizing (pamphlet)](https://docs.wixstatic.com/ugd/8246de_248a47ddbd3c4d209667ff5508ee300b.pdf) ([full book](https://leanpub.com/patterns-for-decentralised-organising/)), [IDOARRT](https://toolbox.hyperisland.com/idoarrt-meeting-design), [9 Artifacts To Seed a Project Team](http://kfjournal.org/wp/index.php/2015/10/19/9-artifacts-to-seed-a-project-team/) and [Pods](https://handbook.enspiral.com/guides/pods.html) and [Stewardship](https://loomio.coop/stewarding.html) from Enspiral.
+
+This is a proposal for how you can grow a Peach. A Peach is a form of working group which together built up the European Scuttlebutt Consortium tree. Feel free to hack!
+
+1. **Compassion Building** - Make time to get to know each other (more)! Some ways to do this: Tell your life story to each other through life charts of highs and lows. Have meetings where you talk about other things than what you usually work with. Distribute care labour and strengthen the team's collective emotional intelligence. Set up a peer-to-peer support system like Stewardship (defined in one of the links above).
+
+2. **Interaction Format** - Talk about and save the conclusions for the future:
+	1. Set an intention and desired ourcome for your Peach.
+	2. Communicate expectations. How do you plan to contribute? How much time and with what kind of work scope? Be as specific as possible to help avoid missunderstandings or boundary crossings. 
+	3. Decide how you will work together / when. Async/Meeting dates/Schedules etc.
+	4. Decide how you will communicate. Where will you communicate? What are the expectations of response time?
+	5. Action Item Tracking System - How do we track what needs to be done?
+
+3. **Culture**
+	1. Rituals - are practices that help boost the interactivity of your calls. It could be as simple as holding a minute of silence at the start of the call.
```

Comment:
```suggestion
	1. Rituals - Practices that help boost the interactivity of your calls. It could be as simple as holding a minute of silence at the start of the call.
```

Reply:
yyy